sdk/storage/azure-storage-blob-changefeed/azure/storage/blobchangefeed/_models.py

```diff
@@ -0,0 +1,294 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+# pylint: disable=too-few-public-methods, too-many-instance-attributes
+# pylint: disable=super-init-not-called, too-many-lines
+import collections
+import json
+from datetime import datetime
+from io import BytesIO
+
+from avro.datafile import DataFileReader
+from avro.io import DatumReader
```

Comment:
I thought we were using the new internal avro parser?

Reply:
Yeah, at that time internal parser wasn't ready, it's switched to internal parser now!